values/build.gradle

```diff
@@ -0,0 +1,58 @@
+
+buildscript {
+    configurations.all {
+        resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
+    }
+    dependencies {
+        classpath group: 'org.spine3.tools', name: 'proto-lookup-plugin', version: '1.1', changing: true
+    }
+}
+
+dependencies {
+    compile group: 'io.grpc', name: 'grpc-all', version: '0.13.0'
+    testCompile project(path: ":client", configuration: 'testArtifacts')
```

Comment:
We will have another round of dependency analysis after this PR is closed. We may want to have a mini-core, which would form the basis for compiler and values sub-project.